apps/api/src/schema/library-traffic.ts

```diff
@@ -3,18 +3,39 @@ import { z } from "@hono/zod-openapi";
 export const libraryTrafficQuerySchema = z.object({
   locationName: z
     .string()
+    .describe("Filter results by exact location name")
```

Comment:
convention is to use `description` property of param to `openapi` (as you do below)